ACE/include/makeinclude/platform_android.GNU

```diff
@@ -9,7 +9,123 @@ no_hidden_visibility ?= 1
 # as of NDK r6 inlining is required
 inline ?= 1
 
-include $(ACE_ROOT)/include/makeinclude/platform_linux_common.GNU
+# start of: include most of platform_linux_common.GNU
+
+# We always include config-linux.h on Linux platforms.
+ACE_PLATFORM_CONFIG ?= config-linux.h
```

Comment:
I've confirmed that this gets overwritten elsewhere and doesn't get included. I'll remove it too.